dashboard/test/ui/runner.rb

```diff
@@ -313,7 +320,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if $options.log_dev
+      HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    end
```

Comment:
My concern is that no one would use this flag if it is optional, and the goal of reducing chatter in the Developers room would not be met. I've reverted this part of the change for now so we can get the localhost stuff in.